packages/catalog-client/api-report.md

```diff
@@ -34,6 +34,10 @@ export interface CatalogApi {
     request?: GetEntitiesRequest,
     options?: CatalogRequestOptions,
   ): Promise<GetEntitiesResponse>;
+  getEntitiesByRef(
```

Comment:
So, maybe it's good to rename this into `getEntitiesByRefs` to make it consistent to the API route?